AmplifyPlugins/Analytics/Sources/AWSPinpointAnalyticsPlugin/AWSPinpointAnalyticsPlugin+Options.swift

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+extension AWSPinpointAnalyticsPlugin {
+    public struct Options {
+        public static let defaultAutoFlushEventsInterval: UInt = 60
+        public static let defaultTrackAppSession = true
+        public static let defaultAutoSessionTrackingInterval: UInt = {
+        #if os(macOS)
+            .max
+        #else
+            5
+        #endif
+        }()
+
+        public let autoFlushEventsInterval: UInt
```

Comment:
Yes, the intention was to move the internal `Int` to `UInt` so it can't be negative. The API discussion was geared towards where we want to place the "options" type, without much focus on the property naming and types themselves. Will look at this further

Reply:
Updated PR description and PR to use TimeInterval